app/services/ocean_service.rb

```diff
@@ -0,0 +1,62 @@
+class OceanService
+  ISLAND_SCATTER_RADIUS = 200
+  CENTER = [500,500]
+
+  def initialize(number_of_islands)
+    @number_of_islands = number_of_islands
+  end
+
+  def call
+    { islands: islands }
+  end
+
+  def islands
+    if @number_of_islands.zero? || @number_of_islands.nil?
```

Comment:
also the `#.nil?` check is not needed here I think, since we controll what values go into the service.
If at all we would raise an exception when this is nil. 
like `raise ArgumentError.new('number of islands missing')`